lua/wire/gpulib.lua

```diff
@@ -96,7 +101,7 @@ if CLIENT then
 		for i, RT in pairs( RenderTargetCache ) do
 			if not RT[1] and  RT[2] == false then -- not used and doesn't exist, let's create the render target.
 
-					local rendertarget = GetRenderTarget("WireGPU_RT_"..i, 512, 512)
+					local rendertarget = GetRenderTargetEx("WireGPU_RT_"..i, 512, 512, 0, 0, 0, 0, IMAGE_FORMAT_RGBA8888)
```

Comment:
What's the point of adding more complication to the rendertarget creation? The default works just as well if not better